device/peripherals/modules/sht25/manager.py

```diff
@@ -21,6 +21,9 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         self.temperature_name = self.variables["sensor"]["temperature_celsius"]
         self.humidity_name = self.variables["sensor"]["humidity_percent"]
 
+        # Only need to update once a minute, the default of 5 secs is too often
```

Comment:
Is this for the LHTC in general, or should we be slowing down how often we look at this temp on the PFCs as well?

Reply:
After off line conversation, this makes sense for all devices. Plants don't react that fast, so 5 seconds is way too fast...